reconfiguration/cmf/concord.cmf

```diff
@@ -60,6 +60,19 @@ Msg GetVersionCommand 19 {
   bytes place_holder
 }
 
+Msg InstallCommand 20 {
+    string version
+}
+
+Msg InstallStatusCommand 21 {
+    string version
```

Comment:
If we make the assumption that only one release version is being installed at a time,`version` can be made optional for the `InstallStatusCommand`. 
The main use-case: If we want to check at the operator if any installation is in progress before starting a new install, we should be able to run the install status command without a version argument.

Reply:
done